include/dxc/Support/dxcapi.extval.h

```diff
@@ -30,24 +30,19 @@ class DxcDllExtValidationLoader : public DllLoader {
   to satisfy the IDllLoader interface. The parameter is ignored
   because the relevant dlls are specific and known: dxcompiler.dll
   and dxil.dll. This class is not designed to handle any other dlls */
-  HRESULT OverrideDll(LPCSTR dll, LPCSTR entryPoint) override {
+  HRESULT OverrideDll(LPCSTR dll, LPCSTR entryPoint) {
```

Comment:
It should be possible to remove these functions that are no longer a part of the interface from `DxcDllExtValidationLoader` now.
* `OverrideDll`
* `HasCreateWithMalloc`
* `Detach`